SitumWayfinding/Classes/Positioning/PositioningViewController.swift

```diff
@@ -261,8 +262,30 @@ class PositioningViewController: UIViewController, GMSMapViewDelegate, UITableVi
         }
         
         let camera = GMSCameraPosition.camera(withLatitude: latitude, longitude: longitude, zoom: zoom)
+        // Check if values are correct (!= -1). Otherwise it could break the app.
+        var zoomValues = minMaxZoomValues()
+
+        if zoomValues.minZoom >= 0 && zoomValues.maxZoom >= 0 {
```

Comment:
Esta comprobación realmente estase facendo na función de abaixo `minMaxZoomValues` non? Creo que se podería facer un if antes do retorno (na función minMaxZoomBalues) para imprimir a mensaxe "Unable to set min and maxZoom. Leaving default values." directament alí, e así deixar esta función máis limpa

Reply:
Se algun dos dous cumple os requisitos ten sentido non aplicar ningun? Comportase igual en Android?

Reply:
Por outro lado vai entrar no segundo else? xa que na función minMaxZoom 
faise
if minZoom <= 0 -> minZoom = kGMSMinZoomLevel
co cal a comprobación de que o minZoom>0 sera sempre true
e o mesmo para o maxZoom
Non?

Reply:
Efectivamente vendo o código parece que o valor nunca será < 0 porque a función minMaxZoomValues evítao. Entendo que ese if sobra.
En android sucede algo parecido, pero temos máis problemática. O caso é que o facemos por separado, pero ten en conta que nós temos dous métodos diferentes e en iOS só hai 1 (setMinZoomPreference & setMaxZoomPreference vs mapView.setMinZoom(min, max)).